common/lib/container-definitions/src/audience.ts

```diff
@@ -12,7 +12,7 @@ import { IClient } from "@fluidframework/protocol-definitions";
 export interface IAudience extends EventEmitter {
 
     on(event: "addMember", listener: (clientId: string, details: IClient) => void): this;
-    on(event: "removeMember", listener: (clientId: string) => void): this;
+    on(event: "removeMember", listener: (clientId: string, details: IClient | undefined) => void): this;
```

Comment:
Can you please give it better name? i.e. "details" -> client / member
Same for addMemember